heudiconv/cli/run.py

```diff
@@ -248,11 +248,11 @@ def process_args(args):
 
     outdir = op.abspath(args.outdir)
 
-    import etelemetry
     try:
+        import etelemetry
```

Comment:
this can be outside, unless etelemetry is not a dependency.

Reply:
It could be. But if it fails to import for some reason - whole process crashes. on Debian we don't have it yet, I have to patch - this would about needing to patch. So IMHO a little cost to ensure absent side effects and ease packaging where it is not available yet